app/_data/tables/version_errors_konnect.yml

```
@@ -3019,4 +3040,54 @@ messages:
       Remove any `sni` elements with unsupported wildcard formats, or upgrade to Kong
       version >= 3.7.0.0.
     DocumentationURL: |
-      [Route entity](/gateway/api/admin-oss/latest/#/Routes)
+      [Route entity](/gateway/latest/admin-api/#route-object)
```

```suggestion
      [Route entity](/gateway/api/admin-oss/latest/#/Routes)
```
This link doesn't exist anymore.